README.md

```diff
@@ -1,2 +1,6 @@
 # kiishare-api
 Backend  to support keyboard config sharing service
+
+## Setup
+
+1. create tables `pipenv run python -c 'import model; model.setup_db()`
```

Comment:
Doesn't say anything about running the server though, is it just `python kllawesome.py`?

Reply:
I'd also add which version of python we're using

Reply:
Running the server is done by `FLASK_APP=kllawesome.py flask run`. I'm sure that there is a better way of doing it but it works for now in dev.